src/loginServices.js

```diff
@@ -161,6 +161,31 @@ export function getLocale(okapiUrl, store, tenant) {
     });
 }
 
+export function getUserLocale(okapiUrl, store, tenant, userId) {
+  if (userId) {
+    return fetch(`${okapiUrl}/configurations/entries?query=(configName==localeSettings and userId==${userId})`,
```

Comment:
Could we actually wrap this in a `hasPerm` so we don't nail the console with an error on login?

Reply:
Practically speaking, if a user has permission to do anything then they're gonna end up with config-read permission, so I think the check is unlikely to fail in real-world scenarios. That said, it's  obviously The Right Thing to do here, and for all these on-login fetches.

Reply:
Stashed the check in `loadResources()`, which calls `getUserLocale()` et al.